tests/e2e/sideNav-dropdown-test.js

```diff
@@ -0,0 +1,129 @@
+import createServer from '../createServer'
+import { Selector, ClientFunction } from 'testcafe'
+// import { ReactSelector, waitForReact } from 'testcafe-react-selectors'
+require('dotenv').config()
+
+const getWindowLocation = ClientFunction(() => window.location)
+
+const hostname = `localhost`
+const port = 3002
+const baseUrl = `${hostname}:${port}`
+
+let server
+
+const openSideNav = async t => {
+  const hamburger = await Selector('[class^="Hamburger"')
+  await t
+    .resizeWindowToFitDevice('iPhone 6', {
+      portraitOrientation: true
+    })
+    .click(hamburger)
+}
+
+fixture`Side Nav Menu`.page`${baseUrl}`
+  .before(async () => {
+    server = createServer(port)
+  })
+  //   .beforeEach(async () => {
+  //     await waitForReact()
+  //   })
+  .after(() => server.close())
+
+test('we are on the homepage', async t => {
+  const location = await getWindowLocation()
+  await t.expect(location.href).contains(baseUrl)
+})
+
+test('a hamburger is present on the page', async t => {
+  const hamburger = await Selector('[class^="Hamburger"').filterVisible()
+  await t
+    .resizeWindowToFitDevice('iPhone 6', {
+      portraitOrientation: true
+    })
+    .expect(hamburger.exists)
+    .ok()
+})
+
+test('the side nav is not expanded at its initial state', async t => {
+  const sideNavPanel = Selector('nav[class^="SideNav"').withAttribute('open')
+  await t.expect(sideNavPanel.exists).notOk()
+})
+
+test('when the hamburger button is clicked the side nav expands', async t => {
+  await openSideNav(t)
+
+  const sideNavPanel = Selector('nav[class^="SideNav"').withAttribute('open')
+  await t.expect(sideNavPanel.exists).ok()
+})
+
+test('clicking a side Nav link redirects to the correct url and closes the side Nav', async t => {
+  await openSideNav(t)
+  const sideNavPanel = Selector('nav[class^="SideNav"]').withAttribute('open')
+
+  const joinUsLink = sideNavPanel.find('a').withText('Join us')
+  await t.click(joinUsLink)
+
+  const location = await getWindowLocation()
+  await t.expect(location.href).contains(`${baseUrl}/join-us`)
+  await t.expect(sideNavPanel.exists).notOk()
+})
+
+test('when the side nav expands for the first time no sub-item is shown by default', async t => {
```

Comment:
Don't we auto expand dropdowns if the active page is within the dropdown?

Reply:
Yes, but this example starts at `baseUrl` (line 34).

Reply:
I was more referring to the name of the test. If anyone comes to the project should be able to garner a bit information on behaviour from tests. The test name should reflect that we don't open any dropdowns if none of the dropdowns contain links to the current path, we should also have a test to reflect the opposite